package.json

```diff
@@ -57,6 +57,56 @@
           "default": "off",
           "description": "Traces the communication between VS Code and the XML language server.",
           "scope": "window"
+        },
+        "xml.catalogs": {
+          "type":"array",
+          "default": [],
+          "description": "Catalog stuff",
+          "scope": "window"
+        },
+        "xml.logs.client": {
+         "type": "boolean",
+         "default": true 
+        },
+        "xml.logs.file": {
```

Comment:
description?

Reply:
not sure we should actually expose that

Reply:
So only use the default path from extensions.js?